src/trace_processor/importers/ftrace/ftrace_parser.cc

```diff
@@ -4254,4 +4258,15 @@ void FtraceParser::ParseMaliGpuPowerState(int64_t ts,
       context_->track_tracker->InternTrack(kMaliGpuPowerStateBlueprint);
   context_->event_tracker->PushCounter(ts, event.to_state(), track);
 }
+
+void FtraceParser::ParseDmabufRssStat(int64_t ts,
+                                      uint32_t pid,
+                                      ConstBytes blob) {
+  protos::pbzero::DmabufRssStatFtraceEvent::Decoder evt(blob);
+  UniqueTid utid = context_->process_tracker->GetOrCreateThread(pid);
+  context_->event_tracker->PushProcessCounterForThread(
+      EventTracker::RssStat{"rss.dmabuf"}, ts, static_cast<double>(evt.rss()),
```

Comment:
I'm not a huge fan of making this the same as rss_stat honestly. People will feel like "this is connected with the rss_stat event" when it is a different event

I would suggest instead making it clear that it's a different event in both the name of the track, the enum for pushprocess counter and how these are handled in the UI.